jest.config.ts

```diff
@@ -183,7 +185,7 @@ export default {
   // A map from regular expressions to paths to transformers
   transform: {
     "^.+\\.(js|jsx|ts|tsx)$": [
-      "esbuild-jest",
+      "esbuild-jest2",
```

Comment:
I was unable to use the `satisfies` keyword without esbuild-jest blowing up on me.  Turns out it's been unmaintained for 5 years.  (`esbuild-jest2` doesn't look that much better; maybe we actually move to `swc-jest` or something?)

Reply:
Ok, `esbuild-jest2` then blew up when running with the `ci` flag, for some reason (but not regular unit tests).  `swc-jest` was too much slower - the themePalette tests in particular took like a minute in ci mode.  There were like 3-4 other esbuild-jest transformers which were all broken in pretty similar ways, so I vendored this one and think I've fixed the issue (it was literally a patch to fall back to babel when encountering calls to `jest.mock`, which maybe esbuild used to struggle with, but then babel wasn't happy with things.  Seems like esbuild works fine now, at any rate?).

Reply:
Nevermind, I was fooled by the jest cache causing tests to pass when they shouldn't have.  Uncommented out the babelTransform case, but removed the part that sent it down that path when `opts.instrument` was true (which `unit-ci` sets but `unit` doesn't).  This still seems a bit fragile and maybe relies on test code which has a mock in it not importing anything that imports a file with a `satisfies` keyword?